gym/wrappers/transform_reward.py

```diff
@@ -1,29 +1,41 @@
+"""Wrapper for transforming the reward."""
+from typing import Callable
+
+import gym
 from gym import RewardWrapper
 
 
 class TransformReward(RewardWrapper):
-    r"""Transform the reward via an arbitrary function.
-
-    Example::
+    """Transform the reward via an arbitrary function.
```

Comment:
I don't think many people use `reward_range` but I think this warning is still necessary.
This also plays into your previous comment: It might be sensible to have the constructor of `RewardWrapper` set the new `reward_range` to (-inf, inf) by default.

```suggestion
    """Transform the reward via an arbitrary function.
    
    Warning:
        If the base environment specifies a reward range which is not invariant under :attr:`f`, the :attr:`reward_range` of the wrapped environment will be incorrect.
```